native/core/src/models.rs

```diff
@@ -501,6 +512,37 @@ impl TryFrom<AnyActerEvent> for AnyActerModel {
                     reason: r.unsigned.redacted_because,
                 }),
             },
+
+            AnyActerEvent::Reaction(e) => match e {
+                MessageLikeEvent::Original(m) => Ok(AnyActerModel::Reaction(m.into())),
+                MessageLikeEvent::Redacted(r) => Err(Error::ModelRedacted {
+                    model_type: ReactionEventContent::TYPE.to_owned(),
+                    meta: EventMeta {
+                        room_id: r.room_id,
+                        event_id: r.event_id,
+                        sender: r.sender,
+                        origin_server_ts: r.origin_server_ts,
+                    },
+                    reason: r.unsigned.redacted_because,
+                }),
+            },
+
+            AnyActerEvent::Redaction(e) => {
+                info!("AnyActerEvent::Redaction - {:?}", e);
+                match e {
+                    RoomRedactionEvent::Original(m) => Ok(AnyActerModel::Redaction(m.into())),
```

Comment:
Okay, now I am confused. what is that RedactionEvent if we already have a `MessageLikeEvent::Redacted`? Why is that even needed?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/d50b2841f52227a6a85d51b8dc2d0fe16464a7f4